docs/USAGE-GUIDE.md

```diff
@@ -419,17 +423,22 @@ The "default" and "default_tls13" version is special in that it will be updated
 
 The "rfc9151" security policy is derived from [Commercial National Security Algorithm (CNSA) Suite Profile for TLS and DTLS 1.2 and 1.3](https://datatracker.ietf.org/doc/html/rfc9151).
 
-s2n-tls does not expose an API to control the order of preference for each ciphersuite or protocol version. s2n-tls follows the following order:
+s2n-tls does not expose an API to control the order of preference for each ciphersuite or protocol version. Unless [ChaCha20-Poly1305 boosting](#chacha20-poly1305-boosting) is enabled then s2n-tls follows the following order:
 
```

Comment:
This order isn't for run-time cipher selection logic, it's for the general order cipher suites are listed in security policies. I would not mention chacha20 boosting here.